.github/workflows/uat-deploy-java-contracts.yml

```diff
@@ -0,0 +1,17 @@
+name: Deploy Contracts After PR Merge to main branch
+
+on:
+  push:
+    branches:
+      - "main"
+
+jobs:
+  deploy-contracts-uat:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Deploy all contracts on UAT environment
+        run: cd contracts/javascore && ./gradlew clean build && ./gradlew optimizedJar && ./gradlew deployToUat -PdeploymentENV=uat
```

Comment:
let's use a different keystore than the godwallet for our uat deployment as it still needs to be authorized.

Reply:
we can keep the keystore in the git secrets as json file

Reply:
Have separated out the keystore and password from coderepo. And have used different keystore for uat deployment.